crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -834,6 +735,147 @@ impl ScheduleGraph {
         }
     }
 
+    /// Adds the config nodes to the graph.
+    ///
+    /// `process_config` is the function which processes each config node and returns a corresponding `NodeId`.
+    ///
+    /// The fields on the returned [`ProcessConfigsResult`] are:
+    /// - `nodes`: a vector of all node ids contained in the nested `NodeConfigs`
+    /// - `densely_chained`: a boolean that is true if all nested nodes are linearly chained (with successive `after` orderings) in the order they are defined
+    #[track_caller]
+    fn process_configs<T: ProcessNodeConfig>(
```

Comment:
Weird, I wonder why the syntax highlighter broke here.

Reply:
Could it be the attribute?
I've had that happen on code near attributes a few times.